javav2/example_code/ec2/src/main/java/com/example/ec2/Route53Scenario.java

```diff
@@ -0,0 +1,394 @@
+                System.out.println("Renewal: "+pr.renewalPrice().price() + " " +pr.renewalPrice().currency());
+                System.out.println("Transfer: "+pr.transferPrice().price() + " " +pr.transferPrice().currency());
+                System.out.println("Transfer: "+pr.transferPrice().price() + " " +pr.transferPrice().currency());
+                System.out.println("Change Ownership: "+pr.changeOwnershipPrice().price() + " " +pr.changeOwnershipPrice().currency());
+                System.out.println("Restoration: "+pr.restorationPrice().price() + " " +pr.restorationPrice().currency());
+                System.out.println(" ");
+            }
+
+        } catch (Route53Exception e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
+    }
+
+
+    public static void listBillingRecords(Route53DomainsClient route53DomainsClient) {
+        try {
+            Date currentDate = new Date();
+            LocalDateTime localDateTime = currentDate.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
+            ZoneOffset zoneOffset = ZoneOffset.of("+05:30");
+            LocalDateTime localDateTime2 = localDateTime.minusYears(1);
+            Instant myStartTime = localDateTime2.toInstant(zoneOffset);
+            Instant myEndTime = localDateTime.toInstant(zoneOffset);
+
+            ViewBillingRequest viewBillingRequest = ViewBillingRequest.builder()
+                .maxItems(10)
+                .start(myStartTime)
+                .end(myEndTime)
+                .build();
+
+            ViewBillingResponse response = route53DomainsClient.viewBilling(viewBillingRequest);
+            List<BillingRecord> records = response.billingRecords();
+            if (records.isEmpty()) {
+                System.out.println("\tNo billing records found for the past year.");
+            } else {
+                for (BillingRecord record : records) {
+                    System.out.println("Bill Date: "+record.billDate());
+                    System.out.println("Operation: "+ record.operationAsString());
+                    System.out.println("Price: "+record.price());
+                }
+            }
+
+        } catch (Route53Exception e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
+
+    }
+
+    public static void listOperations(Route53DomainsClient route53DomainsClient) {
+        try {
+            Date currentDate = new Date();
+            LocalDateTime localDateTime = currentDate.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
+            ZoneOffset zoneOffset = ZoneOffset.of("+05:30");
+            localDateTime = localDateTime.minusYears(1);
+            Instant myTime = localDateTime.toInstant(zoneOffset);
+
+            ListOperationsRequest operationsRequest = ListOperationsRequest.builder()
+                .submittedSince(myTime)
+                .maxItems(10)

```

Comment:
Same as above on zone offset and paging.

Reply:
Same comment!  I used "US/Eastern" here too